cognite/_utils.py

```diff
@@ -157,13 +187,12 @@ def _print_progress(self):
         sys.stdout.flush()
 
     def _get_start_end(self, tag_ids, start, end):
-        from cognite.timeseries import get_latest
         first_timestamp = float("inf")
         for tag in tag_ids:
             if isinstance(tag, str):
-                first_datapoint = _get_first_datapoint(tag, start, end, self.api_key, self.project)
+                first_datapoint = get_first_datapoint_ts(tag, start, end, self.api_key, self.project)
             else:
-                first_datapoint = _get_first_datapoint(tag['tagId'], start, end, self.api_key, self.project)
+                first_datapoint = get_first_datapoint_ts(tag['tagId'], start, end, self.api_key, self.project)
```

Comment:
maybe make 'tagId' as a constant? It will be changed in the new public API version. They are trying to get away from 'tagId' concept.